Dockerfile

```diff
@@ -12,18 +12,17 @@ RUN cd $GOPATH/src/github.com/containers/skopeo \
 #---------------------------------------------------------------------
 # STAGE 2: Build the kubernetes-monitor
 #---------------------------------------------------------------------
-FROM node:erbium-alpine
+FROM registry.access.redhat.com/ubi8/ubi:latest
 
 LABEL maintainer="Snyk Ltd"
 
 ENV NODE_ENV production
 
-RUN apk update
-RUN apk upgrade
-RUN apk --no-cache add dumb-init
+RUN curl -sL https://rpm.nodesource.com/setup_12.x | bash -
+RUN yum install -y nodejs
```

Comment:
can't we somehow install dumb-init?

Reply:
dumb-init is a drop-in for when an init system doesn't exist, `ubi` has one already:
https://github.com/snyk/kubernetes-monitor/pull/429#discussion_r403958309